README.rst

```diff
@@ -146,20 +146,49 @@ non-universal rules that match the URL, the universal ones are applied.
 Configuration
 =============
 
-``duplicate-url-discarder`` uses the following Scrapy settings:
+**duplicate-url-discarder** uses the following Scrapy settings:
 
-``DUD_LOAD_RULE_PATHS``: it should be a list of file paths (``str`` or
-``pathlib.Path``) pointing to JSON files with the URL rules to apply:
+* ``DUD_LOAD_RULE_PATHS``: it should be a list of file paths (``str`` or
+  ``pathlib.Path``) pointing to JSON files with the URL rules to apply:
 
-.. code-block:: python
+  .. code-block:: python
 
-    DUD_LOAD_RULE_PATHS = [
-        "/home/user/project/custom_rules1.json",
-    ]
+      DUD_LOAD_RULE_PATHS = [
+          "/home/user/project/custom_rules1.json",
+      ]
```

Comment:
Would it be helpful to explain how to use locally defined rules with spiders running in Scrapy Cloud?

Reply:
Yeah, this PR doesn't change this part of the doc except formatting but it's nice to have as a separate PR.

Reply:
Thanks for raising @PyExplorer . Created https://github.com/zytedata/duplicate-url-discarder/issues/19 for tracking.